src/plugin/plugin.rs

```diff
@@ -128,5 +174,9 @@ impl<PhysicsHooksData: 'static + WorldQuery + Send + Sync> Plugin
                 (),
             )));
         }
+
+        if app.world.get_resource::<RapierConfiguration>().is_none() {
+            app.insert_resource(RapierConfiguration::default());
+        }
```

Comment:
Why we check for `RapierConfiguration` now? Previously it was inserted unconditionally.

Reply:
It is necessary for user configuration.  As mentioned in the top-level PR comment, in order for the rollback plugin to work deterministically, the user must also [insert a configuration](https://github.com/cscorley/bevy_ggrs_rapier_example/blob/59681f24fb989203dbf74dab68f1ed4e450d44ea/src/main.rs#L85).  The other option is the expose the TimestepMode as a plugin configuration step (e.g. a `fn with_timestep_mode`)

Reply:
Gotcha.  I was considering what would be less error-prone for the user and this approach would not be order-dependent.  However, I can swap things around to force the order-dependent insertion/override.

Reply:
https://discord.com/channels/691052431525675048/742884593551802431/973641147912179792

A user had problem with that, as inserting a resource before creating a plugin is sometimes used, it might be a good idea to keep this change (maybe in another PR to discuss it further ?)